src/parsers/utils/fortran.js

```diff
@@ -0,0 +1,110 @@
+import { regex } from "./settings";
```

Comment:
This file doesn't need to be in utils, can be one level above

Reply:
Are you talking about `fortran.js` or `settings.js`?